Update rockchip.md add (#1749)
jellyfin/jellyfin.org: docs/general/post-install/transcoding/hardware-acceleration/rockchip.md

3. Enable RKMPP in Jellyfin and uncheck the unsupported codecs.

#### Docker Compose exampe

> Root permission is NOT required - Setps 2 ans 3 from Linux Setup above are required

Reviewer: How I see it, those are only not required, because you added the needed Paths manually.
This is fine, however should probably be the same between the docker and docker-compose examples.

Author: not the case, the added two new lines in relation with app-armor and security are needet to run docker in rootless mode. The user is added to video an render group - but thats not enough. Old howto add elevatet privileges always.

> Root permission is NOT required - Setps 2 ans 3 from Linux Setup above are required

Reviewer 2: ```suggestion
Root permission is NOT required - Steps 2 and 3 from Linux Setup above are required
```

docker-compose.yml look like this:

:::

> ```shell

Reviewer: ```suggestion
```yaml
```

this SHOULD work just fine (actually I am not sure though, so dont quote me on that 😆)

services:
  jellyfin:
    container_name: jellyfin
    image: jellyfin/jellyfin:latest # ORIGINAL
    security_opt: # enables full access to /sys and /proc not need privileged: true
      - systempaths=unconfined
      - apparmor=unconfined

>     user: 1000:100   

Author: user is in render an video group - but mpp service need rights in /sys

    environment:
      TZ: Europe/Berlin # Your timezone
      TMPDIR: /config/temp
    volumes:
      - /mnt/ssd/jellyfin/config:/config                 # Your config path
      - /mnt/ssd/jellyfin/cache:/cache                   # Your cache path
      - /mnt/jellyfin_external_hdd/media:/media          # Your media path
    ports:
      - 8096:8096       # Web UI - you need this
      - 8920:8920       # HTTPS
      - 7359:7359/udp   # DIA

>       - 1900:1900/udp   # DLNA

Author: its from original how to - if forwading of multicast will not work we should remove the DLNA line

>       - 1900:1900/udp   # DLNA

Reviewer: Yes, which is why I am pointing it out here.
I also added an explainaition for this to the DLNA page on our docs:
https://jellyfin.org/docs/general/post-install/networking/dlna/#general

    restart: unless-stopped
    devices:

>      # Rock 5 ITX (RK3588) GPU devices for hardware transcoding

Reviewer 2: We should avoid mentioning the SBC vendor name. The Docker config is vendor agnostic.
```suggestion
     # Rockchip VPU and GPU devices for hardware transcoding
```

>       - /dev/dri:/dev/dri                 # DRM/KMS for VAAPI

Reviewer 2: There's no VAAPI on Rockchip:
```suggestion
      - /dev/dri:/dev/dri                 # DRM Dumb Buffer allocator
```

      - /dev/mali0:/dev/mali0             # Mali GPU

>       - /dev/mpp_service:/dev/mpp_service # Rockchip MPP for hw encoding

Reviewer 2: ```suggestion
      - /dev/mpp_service:/dev/mpp_service # Rockchip MPP for HW transcoding
```

      - /dev/rga:/dev/rga                 # Rockchip RGA (2D graphics)

>       - /dev/dma_heap:/dev/dma_heap       # CMA Memory - be sure to set at least 256MB of CMA at Kernel-Boot-Parameter

Reviewer 2: Rockchip SoCs have IOMMUs and do not use CMA:

```suggestion
      - /dev/dma_heap:/dev/dma_heap       # DMA_HEAP allocator
```
